packages/strapi-plugin-users-permissions/admin/src/containers/AuthPage/styles.scss

```diff
@@ -91,3 +97,22 @@
     height: 34px;
   }
 }
+
+.buttonForgotSuccess {
+  border: 1px solid #5A9E06;
+  color: #5A9E06;
+}
+
+.forgotSuccess {
+  width: 100%;
+  // margin-top: -2px;
+  text-align: center;
+  color: #5A9E06;
+  font-size: 13px;
+  font-weight: 500;
+  > p {
```

Comment:
You forgot the `font-weight: 500;` property to display the email with semibold weight.